build_docker.sh

```diff
@@ -1 +0,0 @@
-docker build -t mottosso/qt.py27 -f Dockerfile-py2.7 .
```

Comment:
I removed the build part from the README.md just because it's probably something users won't be doing. Instead they'll just `docker run` and the image is pulled.

I've added the `build_docker.sh` back but I haven't added the build command back to the main `README.md`. Let me know how you wish we do this. Either way is fine by me.